src/App.jsx

```diff
@@ -1,15 +1,30 @@
+// import { props } from 'cypress/types/bluebird';
 import React from 'react';
 import './App.scss';
+import { Sum } from './components/Sum';
 
 const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <Sum
+      a={2}
+      b={5}
+    />
+    <Sum
+      a={-5}
+      b={5}
+    />
+    <Sum
+      a={10}
+      b={0}
```

Comment:
Zero props should be skipped.
![image](https://user-images.githubusercontent.com/60885495/171364147-2884c429-3e8a-4dea-97ce-5b40bdd6482b.png)

Reply:
Fixed.